diff --git a/setup.cfg b/setup.cfg
index aafa110..4dcf73a 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,8 +1,8 @@
 [_ldap]
-library_dirs = /usr/lib /usr/lib64 /usr/local/lib /usr/local/lib64
-include_dirs = /usr/include /usr/include/sasl /usr/local/include /usr/local/include/sasl
+library_dirs = /usr/lib /usr/lib64
+include_dirs = /usr/include/sasl /usr/include
 defines = HAVE_SASL HAVE_TLS HAVE_LIBLDAP_R
-extra_compile_args = 
+extra_compile_args = -g
 extra_objects = 
 libs = ldap_r
 
